TestCases/strayalien/0058-feel-number-function/0058-feel-number-function.dmn

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="http://www.montera.com.au/spec/DMN/0058-feel-number-function" name="0058-feel-number-function" id="_i9fboPUUEeesLuP4RHs4vA" xmlns="http://www.omg.org/spec/DMN/20180521/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <description>FEEL built-in function 'modulo(dividend,divisor)' in category numeric functions</description>
+    <decision name="decision001" id="_decision001">
+        <description>Tests FEEL expression: 'number("1.000.000,01", ".", ","))' and expects result: '1000000.01 (number)'</description>
+        <question>Result of FEEL expression 'number("1.000.000,01", ".", ","))'?</question>
+        <allowedAnswers>1000000.01 (number)</allowedAnswers>
+        <variable typeRef="number" name="decision001"/>
+        <literalExpression>
+            <text>number("1.000.000,01", ".", ",")</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision003" id="_decision002">
```

Comment:
I believe a typo.
I believe this is meant to be:
```
<decision name="decision002" id="_decision002">
```

Reply:
thanks.  There will be likely be more of those.  :-) That first PR was 3.5 days effort so it all has been done quite quickly! 

I'll go through them again looking for others.  Thanks again.  I appreciated the feedback.  :-)

Reply:
@tarilabs I've fixed that typo and done a run through all the rest to make sure they line up.  Happy to 'resolve' this now if you are.